distribution/asf-release-process-guide.md

```diff
@@ -71,6 +71,17 @@ Finally, the sample [`docker-compose.yml`](https://github.com/apache/druid/blob/
 
 Once this is completed, open a PR to the master branch. Also, be sure to confirm that these versions are all correct in the release branch, otherwise fix them and open a backport PR to the release branch.
 
+#### Version updates on master and release branch
+
+You also need to update the version manually in some places for both master branch and the release branch.
+- `DRUID_DOCS_VERSION` has to be changed to `0.17.0` in `web-console/src/links.ts`.
+- Docker image tag has to be changed to `0.17.0` for all druid services in `distribution/docker/docker-compose.yml`.
```

Comment:
this part about updating docker-compose.yml is already mentioned https://github.com/apache/druid/blob/master/distribution/asf-release-process-guide.md?plain=1#L62